elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClient.scala

```diff
@@ -175,18 +176,20 @@ class RestlasticSearchClient(endpointProvider: EndpointProvider, signer: Option[
     runEsCommand(query, s"/${index.name}/${tpe.name}/_query", DELETE)
   }
 
-  def startScrollRequest(index: Index, tpe: Type, query: QueryRoot, resultWindow: String = "1m"): Future[ScrollId] = {
+  // Scroll requests have optimizations that make them faster when the sort order is _doc.
```

Comment:
The sort by `_doc` is part of the query. And the client is free to sort by anything basically. `_doc` field is something internal of ES that makes the sorting more efficient. I put another line of comment to call out where to put the sorting. https://github.com/SumoLogic/elasticsearch-client/pull/121/commits/dc6f95ff4b2d8798919efb65d2a667d1d0fa965b